### PR TITLE
Add doc reference for Variable and Parameter

### PR DESCRIPTION
**Pull request by JiayiFeng**

### CODE REVIEW

**Comment by chengduoZH** [comment]
block_
block_2
block_4
block_5
block_6
block_7

**Comment by chengduoZH** [comment]
opt_1
opt_
opt_3
opt_4
opt_5

**Comment by chengduoZH** [comment]
var_1
var_2

File: python/paddle/fluid/framework.py [hunk 1]
```diff
@@ -385,38 +459,9 @@ class Operator(object):
         'channel_recv', 'select', 'gen_nccl_id'
     }
 
-    def __init__(self,
-                 block,
-                 desc,
-                 type=None,
-                 inputs=None,
-                 outputs=None,
+    def __init__(self, block, desc, type, inputs=None, outputs=None,
```
Comment (reyoung) [inline python/paddle/fluid/framework.py]: The API has been changed.

**Review by reyoung — APPROVED** [review]

**Review by skylarch — APPROVED** [review]

**Pull request merged**